numpy/testing/_private/utils.py

```diff
@@ -1484,13 +1490,47 @@ def assert_allclose(actual, desired, rtol=1e-7, atol=0, equal_nan=True,
     When one of `actual` and `desired` is a scalar and the other is
     array_like, the function checks that each element of the array_like
     object is equal to the scalar.
+    This behaviour can be disabled with the `strict` parameter.
 
     Examples
     --------
     >>> x = [1e-5, 1e-3, 1e-1]
     >>> y = np.arccos(np.cos(x))
     >>> np.testing.assert_allclose(x, y, rtol=1e-5, atol=0)
 
+    As mentioned in the Notes section, `assert_allclose` has special
+    handling for scalars. Here, the test checks that the value of `np.sin`
```

Comment:
To make the link render as intended:
```suggestion
    handling for scalars. Here, the test checks that the value of `numpy.sin`
```